jupyterhub/apihandlers/users.py

```diff
@@ -220,11 +220,11 @@ def post(self, name):
         user = self.find_user(name)
         if user is None:
             raise HTTPError(404, "No such user %r" % name)
-        if user.running:
-            # include notify, so that a server that died is noticed immediately
-            state = yield user.spawner.poll_and_notify()
-            if state is None:
-                raise web.HTTPError(400, "%s's server is already running" % name)
+        #if user.running:
```

Comment:
This running check needs to stay. However, it needs to be updated to check for the specified named server, instead  of the user's default server.